WeakAuras/WeakAuras.lua

```diff
@@ -4108,6 +4121,8 @@ function Private.UpdatedTriggerState(id)
       state.changed = false;
     end
   end
+  -- once updatedTriggerStates is complete, and empty states removed, etc., then check for queued trigger_to_custom updates
+  Private.SendQueuedTriggers()
```

Comment:
I don't think it is the correct place to call it, not after ApplyStatesToRegions
In Bufftrigger2 i'd want run it after `UpdateTriggerState` (before Private.UpdatedTriggerState)
In GenericTrigger, after `RunTriggerFunc`, maybe in RunTriggerFunc function at the end?

Reply:
I started out exactly as you say, but that would mean I'm sending the events before WA has cleared up dead states. This would require checking `state.show` to see if it's a real state or not at that point. 

What's the problem with this location?

Reply:
The problem i see with this location is that some code will run twice or more

Reply:
I haven't been able to get the code to run more than required in this location. Can you think of a specific setup that would make this happen so I can test? 
This location feels right to me. I queue things when the triggers update (RunTriggerFunc and UpdateTriggerState) but wait until WA cleans up states table before sending the events. Things could be queued more than once but that won't make anything fire anyway.